README.rst

```diff
@@ -348,11 +348,25 @@ TMT_WORKDIR_ROOT
     Path to root directory containing run workdirs. Defaults to
     ``/var/tmp/tmt``.
 
-NO_COLOR
-    Disable colors in the terminal output. Output only plain,
-    non-colored text. See https://no-color.org/ for more
+NO_COLOR, TMT_NO_COLOR
+    Disable colors in the output, both the actual output and
+    logging messages. Output only plain, non-colored text.
+
+    Two variables are accepted, one with the usual ``TMT_``
+    prefix, but tmt accepts also ``NO_COLOR`` to support the
+    NO_COLOR effort, see https://no-color.org/ for more
     information.
 
+TMT_FORCE_COLOR
+    Enforce colors in the output, both the actual output and logging
+    messages. Might come handy when tmt's output streams are not
+    terminal-like, yet its output would be displayed by tools with
+    ANSI color support. This is often the case of various CI systems.
+
+    Note that ``TMT_FORCE_COLOR`` is **stronger** than ``NO_COLOR``
```

Comment:
maybe **preferred** would be a better word than **stronger** ?

Reply:
I think "stronger" is more fitting in this phrasing, one option is stronger, and ~~beats~~ overrides the other one.

Reply:
sounds like some czechizm :) maybe it is just me :)

Reply:
I'd say: If both `TMT_FORCE_COLOR` and `NO_COLOR` are specified, `TMT_FORCE_COLOR` takes priority.

Or, if you want to preserve the sentence: Note that `TMT_FORCE_COLOR` takes priority over `NO_COLOR`.

Reply:
And "Note that `TMT_FORCE_COLOR` takes priority over `NO_COLOR`" it is.